requirements/dev.txt

```diff
@@ -13,3 +13,4 @@ autodocsumm>=0.1.11
 pandas>=0.23.4
 pyparsing==2.2.1  # temporarily fixed to avoid incompatibilities (to be removed)
 docutils==0.17.1  # 0.18 is buggy
+nlopt
```

Comment:
please remove from dev, this should be in bench.txt and imported inline since it's not a core optimizer

Reply:
done, thx